examples/distributed/primitives/example_get_warp.py

```
@@ -11,7 +11,7 @@
 os.environ['NCCL_DEBUG'] = 'WARN'  # silence NCCL log
 
 
-def kernel_(M, num_rank, block_M, threads):
+def get_kernel(M, num_rank, block_M, threads):
```

Ruff (0.12.2)

Unused function argument: `num_rank`

(ARG001)